config/locales/forms/copy_cards_forms/en.yml

```diff
@@ -0,0 +1,20 @@
+en:
+  waste_carriers_engine:
+    copy_cards_forms:
+      new:
+        heading: "Order registration cards for %{reg_identifier}"
+        temp_cards_label: "Number of cards at £5 each"
+        next_button: "Continue"
+        check_the_address: "Check the address to post the cards to:"
```

Comment:
We've got this bit of text, but I can't see it referenced in the view or in the screenshot in the PR description.

Should it be included somewhere?

Reply:
Oh. I have no idea of why it is there :/ I didn't even copy pasted this code! 😮 🤷‍♂ maybe some `sleepcoding`? 😆

Reply:
```suggestion
```

Reply:
Hey @cintamani I've gone back and checked the wireframe, and I think this is a case of the `.yml` was right, you have just missed it off the view.

![Screenshot 2019-12-11 at 13 15 00](https://user-images.githubusercontent.com/1789650/70624529-62c39880-1c18-11ea-8f35-fcf45ceac24a.png)

Unless you are working off a later wireframe ???

Reply:
😮 🤷‍♀ Sorry! I think is just a distraction!